src/domu_cfgs.c

```diff
@@ -0,0 +1,224 @@
+		.machine_dt_compat = DT_PROP(node_id, machine_dt_compat),                          \
+		.mem_kb = DT_PROP(node_id, mem_size) / 1024,                                       \
+		.max_evtchns = DT_PROP(node_id, max_evtchns),                                      \
+		.max_vcpus = DT_PROP(node_id, max_vcpus),                                          \
+		.gic_version = DT_PROP(node_id, gic_version),                                      \
+		.max_maptrack_frames = DT_PROP(node_id, max_maptrack_frames),                      \
+		.gnt_frames = DT_PROP(node_id, gnt_frames),                                        \
+		.cmdline = DT_PROP(node_id, cmdline),                                              \
+                                                                                                   \
+		.flags = (XEN_DOMCTL_CDF_hvm | XEN_DOMCTL_CDF_hap | XEN_DOMCTL_CDF_iommu),         \
+                                                                                                   \
+		.tee_type = DT_PROP(node_id, tee),                                                 \
+                                                                                                   \
+		.load_image_bytes = LOAD_IMAGE_BYTES(node_id),                                     \
+		.get_image_size = GET_IMAGE_SIZE(node_id),                                         \
+		.dtb_start = DTB_START(node_id),                                                   \
+		.dtb_end = DTB_END(node_id),                                                       \
+                                                                                                   \
+		.irqs = (uint32_t[])DT_PROP_OR(node_id, domu_irqs, {}),                            \
+		.nr_irqs = DT_PROP_LEN_OR(node_id, domu_irqs, 0),                                  \
+                                                                                                   \
+		.iomems = (struct xen_domain_iomem[]){GET_IOMEMS(node_id)},                        \
+		.nr_iomems = DT_PROP_LEN_OR(node_id, iomems, 0),                                   \
+                                                                                                   \
+		.dtdevs = (char *[])DT_PROP_OR(node_id, dtdevs, {}),                               \
+		.nr_dtdevs = DT_PROP_LEN_OR(node_id, dtdevs, 0),                                   \
+                                                                                                   \
+		.dt_passthrough = (char *[])DT_PROP_OR(node_id, dt_passthrough, {}),               \
+		.nr_dt_passthrough = DT_PROP_LEN_OR(node_id, dt_passthrough, 0),                   \
+                                                                                                   \
+		.back_cfg = {FILL_DOMU_BACK_CFGS(node_id)},                                        \
+	},
+
+static struct xen_domain_cfg domu_cfg[] = {DT_FOREACH_STATUS_OKAY(xen_domu, FILL_DOMU_CFG_ITEM)};
```

Comment:
const?

Reply:
other configs without const,  I don't know maybe some of fields can be changed i runtime
so, I leave it as is